app/pages/HomePage/components/GuideList/GuideList.jsx

```diff
@@ -82,31 +80,30 @@ const GuideList = () => (
           name="Shelter &#38; Quarantine Updates"
           link="/covid/shelteraccess"
           img={Imgshelteraccess}
-          isTypeform={false}
         />
       </li>
       <li className={styles.item}>
         <GuideCard
           name="Family Homelessness"
           link="https://sheltertech.typeform.com/to/GFEzl2"
           img={ImgFamilyHomelessness}
-          isTypeform
+          categoryId="5"
```

Comment:
i've just hardcoded a random id in for now, until we have answers on https://sheltertech.slack.com/archives/C013NS0057A/p1590026728003000